pkg/controller/apiextensions/composite/reconciler.go

```diff
@@ -39,6 +39,7 @@ import (
 	"github.com/crossplane/crossplane-runtime/pkg/resource/unstructured/composite"
 
 	"github.com/crossplane/crossplane/apis/apiextensions/v1alpha1"
+	composedctrl "github.com/crossplane/crossplane/pkg/controller/apiextensions/composite/composed"
```

Comment:
What's the thinking behind moving the composer into its own sub-package?

Reply:
One reason was that the interfaces/structs would collide with `Composite` resource reconciler, like both having `Configurator` so I'd have to prefix like `ComposedConfigurator`. Also I'd specify the struct name to say it's `Composed*` because they depend on being used for composed resources rather generic things. Then I realized it'd make sense for it to have its own package so that it's easier to see which interface/object is used by what exactly.